platform/posix/transport/include/mbedtls_pkcs11_posix.h

```diff
@@ -0,0 +1,205 @@
+/*
+ * FreeRTOS V202107.00
```

Comment:
Please update other files as well
```suggestion
 * AWS IoT Device SDK for Embedded C 202103.00
```